plutus-core/cost-model/data/models.R

```diff
@@ -374,8 +390,7 @@ modelFun <- function(path) {
         filtered <- data %>%
             filter.and.check.nonempty (fname)  %>%
             discard.overhead ()
-        m <- lm(t ~ pmax(x_mem, y_mem), filtered)
-        adjustModel(m, fname)
+        lm(t ~ pmax(x_mem, y_mem), filtered)
     }
```

Comment:
We *don't* want `return` here.  This is like a nested let-expression whose value is the final line.  If you put `return` here then you return from the enclosing function, which definitely isn't what we want (as I discovered by accident...).